app/models/game.rb

```diff
@@ -1,2 +1,35 @@
 class Game < ApplicationRecord
+  ROUNDS = 10
+  has_many :players, dependent: :destroy
+  has_many :game_dices, dependent: :destroy
+  has_many :rounds, dependent: :destroy
+
+  def setup users
+    # Create all available dice for the game
+    colors = Color.all
+    color_sets = ((users.size * 20.0 + ROUNDS) / colors.size).ceil
```

Comment:
`color_sets` - number of game_dice objects to generate per color in the game based on number of players and number of colors --> so you always have equal game_dice objects per color